app/server/utils/ussd/token_processor.py

```diff
@@ -220,7 +220,7 @@ def fetch_exchange_rate(user: User):
             )
 
     @staticmethod
-    def send_token(sender: User, recipient: User, amount: float, reason_str: str, reason_id: int):
+    def send_token(sender: User, recipient: User, amount: float, reason_str: str = None, reason_id: int = None):
```

Comment:
Tiny change, but can you please change the typing here to:
`reason_str: Optional[str] = None, reason_id: Optional[int] = None`
https://docs.python.org/3/library/typing.html#typing.Optional